### PR TITLE
Implement Spinner

### PR DESCRIPTION
**Pull request by lenchen1112**
Not complete yet. Just engage the Spinner component for me.

### CODE REVIEW

**Comment by tom76kimo** [comment]
good!

**Comment by lenchen1112** [comment]
@tom76kimo Take a look on this!!

**Comment by tom76kimo** [comment]
Great job!

File: src/components/Spinner.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        classString = classnames('mdl-spinner mdl-js-spinner is-upgraded', classObject);
+        return (
+            <div className={classString + ' ' + this.props.className} style={this.props.style}>
+                {this.List.toArray()}
+            </div>
+        );
+    }
+}
+
+Slider.propTypes = {
```
Comment (tom76kimo) [inline src/components/Spinner.js:60]: No~ it's not Slider.

Comment (lenchen1112) [inline src/components/Spinner.js:60]: 0.0 sorry

**Pull request merged**